jcl/src/java.base/share/classes/java/lang/System.java

```diff
@@ -101,6 +104,11 @@
 	private static String platformEncoding;
 	private static String fileEncoding;
 	private static String osEncoding;
+	
+	/*[IF JAVA_SPEC_VERSION >= 11]*/
+	static boolean setErrEncoding;
+	static boolean setOutEncoding;
```

Comment:
These names could be better, rather than leave the reader guessing whether the meaning is 'did set' or 'should set'.